### ISSUE TITLE
Signing: error reported twice and without call stack

### ISSUE THREAD

**Issue by dtivel** [NuGet/Home#6455]
With a trusted package signing certificate and unsigned package execute:

    set NUGET_SHOW_STACK=true
    NuGet.exe sign <PackageFilePath> -CertificatePath <PfxFilePath> -Timestamper http://freetsa.org/tsr

Sample output:
```
Signing package(s) with certificate:
Subject Name: CN=Test certificate for testing NuGet package signing
SHA1 hash: 1D4A1E47FD2E642A00103F73829928816758336A
Issued by: CN=Test certificate for testing NuGet package signing
Valid from: 1/18/2018 9:42:41 AM to 1/19/2018 9:42:41 AM

Timestamping package(s) with:
http://freetsa.org/tsr
NU3024: The timestamp certificate has an unsupported signature algorithm.
NU3024: The timestamp certificate has an unsupported signature algorithm.
```

NU3024 is reported twice when it should be reported only once.

**Comment by mishra14** [comment]
@dtivel was this with detailed verbosity?

**Comment by dtivel** [comment]
That would be by using the `-Verbosity` option, right?  Nope.  The command in the repro steps doesn't use the `-Verbosity` option.

**Comment by mishra14** [comment]
@dtivel I cannot repro this with normal verbosity - 

```
PS > NuGet.exe sign F:\validation\sign\unsigned\nuget.server.3.0.0.nupkg -OutputDirectory F:\validation\sign\signed\ -CertificateFingerprint HASH -Timestamper http://freetsa.org/tsr


Signing package(s) with certificate:
Subject Name: CN=Ankit Mishra
SHA1 hash: HASH
Issued by: ISSUER
Valid from: 7/14/2016 2:09:34 PM to 7/14/2018 2:09:34 PM

Timestamping package(s) with:
http://freetsa.org/tsr
Signed package(s) output path:
F:\validation\sign\signed\
NU3024: The timestamp certificate has an unsupported signature algorithm.

```


Detailed Verbosity - 
```
PS > NuGet.exe sign F:\validation\sign\unsigned\nuget.server.3.0.0.nupkg -OutputDirectory F:\validation\sign\signed\ -CertificateFingerprint HASH -Timestamper http://freetsa.org/tsr -verbosity detailed
NuGet Version: 4.7.0.0


Signing package(s) with certificate:
Subject Name: CN=Ankit Mishra
SHA1 hash: HASH
Issued by: ISSUER
Valid from: 7/14/2016 2:09:34 PM to 7/14/2018 2:09:34 PM

Timestamping package(s) with:
http://freetsa.org/tsr
Signed package(s) output path:
F:\validation\sign\signed\
NU3024: The timestamp certificate has an unsupported signature algorithm.
NU3024: The timestamp certificate has an unsupported signature algorithm.
```

Lets chat, if needed.

**Comment by dtivel** [comment]
It turns out the environment variable `NUGET_SHOW_STACK` was set to `true`.  Updated the repro steps.

Note that there are actually 2 issues here:

1. the duplicate error
1. the stack isn't shown

Both seem low priority given this new repro step.

**Comment by mishra14** [comment]
cc: @PatoBeltran
